docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    image: app:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+    - "8080:8080"
+    environment:
+    - ENV=local
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+  mysql:
+    image:
+      mysql:1.0
```

Comment:
The image tag `mysql:1.0` is incorrect. The official MySQL image tags are typically version numbers like `mysql:5.7` or `mysql:8.0`. Please verify and use a valid MySQL image tag.